src/payment/strategies/adyenv2/adyenv2-payment-strategy.ts

```diff
@@ -315,4 +325,15 @@ export default class AdyenV2PaymentStrategy implements PaymentStrategy {
     private _updateAdyenComponentState(newState: AdyenCardState) {
         this._adyenComponentState = newState;
     }
+
+    private _getBrowserInfo(): BrowserInfoRequest {
+        return {
+            language: navigator.language || navigator.languages[0],
+            color_depth: screen.colorDepth,
```

Comment:
I don't think there's an equivalent property in IE but I guess it's okay to send back `undefined` if it's unavailable.

Reply:
I keep as `screen.colorDepth || 24` 
because : some implementations return 24 for compatibility reasons.
re: https://developer.mozilla.org/en-US/docs/Web/API/Screen/colorDepth